docs/source/getting_started/installation.rst

```diff
@@ -98,6 +98,33 @@ Usage with JupyterHub
 Read the details on our :ref:`JupyterLab on JupyterHub documentation page <jupyterhub>`.
 
 
+Usage with Jupyverse
+--------------------
+
+`Jupyverse <https://github.com/jupyter-server/jupyverse/>`__ is a next-generation Jupyter server based on
```

Comment:
Is it worth adding a short note on limited compatibility of extensions with jupyverse?

Reply:
:+1: